### PR TITLE
[codestyle] Add a default case to this switch

### PR DESCRIPTION
**Pull request by SrijanJha28**
Update XWikiSyntaxChainingRenderer.java by Adding a default case to the switch.

### CODE REVIEW

File: xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxChainingRenderer.java [hunk 1]
```diff
@@ -307,6 +307,8 @@ public void endFormat(Format format, Map<String, String> parameters)
                 break;
             case NONE:
                 break;
+            default : //none of the case matched
```
Comment (tmortagne) [inline xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxChainingRenderer.java]: A more specific comment would be nicer :)

Like "Unsupported format" for example.

Comment (SrijanJha28) [reply xwiki-rendering-syntaxes/xwiki-rendering-syntax-xwiki20/src/main/java/org/xwiki/rendering/internal/renderer/xwiki20/XWikiSyntaxChainingRenderer.java]: ok

**Comment by SrijanJha28** [comment]
done

**Comment by tmortagne** [comment]
Thanks for your contribution !

**Pull request merged**